lib/src/utils.rs

```diff
@@ -55,6 +58,44 @@ pub(crate) fn find_mount_option<'a>(
         .next()
 }
 
+/// Try to (heuristically) determine if the provided path is a mount root.
+pub(crate) fn is_mountpoint(root: &Dir, path: &Utf8Path) -> Result<Option<bool>> {
```

Comment:
Since I wrote this code originally I decided it made sense generically so let's update to use https://github.com/coreos/cap-std-ext/pull/56